Formula/libgosu.rb

```diff
@@ -0,0 +1,46 @@
+class Libgosu < Formula
+  desc "2D game development library"
+  homepage "https://libgosu.org"
+  url "https://github.com/gosu/gosu/archive/v0.10.7.tar.gz"
+  sha256 "31aedcb570a36c344b9a3c12ea13596f994cb52a6fb8c15b8ed83058d9cabe27"
+
+  head do
+    url "https://github.com/gosu/gosu.git"
+  end
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "sdl2"
+
+  def install
+    mkdir "cmake/build"
+    cd "cmake/build" do
```

Comment:
`mkdir "cmake/build" do`. Also, I'd favour just `build` unless this needs to be in the `cmake` directory (no worries if it does).